libs/common/src/types/mongo-db-models/index.ts

```diff
@@ -7,3 +7,4 @@ export * from './users';
 export * from './deletion-audit-logs';
 export * from './tfm-facilities';
 export * from './facility';
+export * from './tfm-deals';
```

Comment:
Do we want to export the deals type too, even though its just a stub?

Reply:
Ooh I don't follow - it's being exported here, is there something else you think should be exported too?

Reply:
Ah I see your comment below about it being in the mongo-db directory

Reply:
Hmm sorry for the rollercoaster of messages, I'm currently using the `TfmDeal` type as a type in `dtfs-central-api/src/repositories/tfm-deals-repo/tfm-deal-cancellation.repo.ts` so using the export in the one place. I think fine to export it for use there?

Reply:
Gotcha, not sure how I didn't read it as such!! Now exporting `Deal` too